Entities/Special/CTF/CTF_Flag.as

```diff
@@ -152,14 +152,14 @@ void onRender(CSprite@ this)
 
 	if (getGameTime() % 15 > 10)
 	{
+		Vec2f base_position = blob.get_Vec2f("flag base position");
+		Vec2f flag_position = blob.getPosition();
```

Comment:
use getInterpolatedPosition when rendering

Reply:
These positionings are not used for rendering, just for determining if we are on the left or right of the origin base.